server/controllers/events/events_controller.go

```diff
@@ -267,223 +390,163 @@ func (e *VCSEventsController) handleAzureDevopsPost(w http.ResponseWriter, r *ht
 	switch event.PayloadType {
 	case azuredevops.PullRequestCommentedEvent:
 		e.Logger.Debugf("handling as pull request commented event")
-		e.HandleAzureDevopsPullRequestCommentedEvent(w, event, azuredevopsReqID)
+		e.HandleAzureDevopsPullRequestCommentedEvent(w, event, azuredevopsReqID, r)
 	case azuredevops.PullRequestEvent:
 		e.Logger.Debugf("handling as pull request event")
-		e.HandleAzureDevopsPullRequestEvent(w, event, azuredevopsReqID)
+		e.HandleAzureDevopsPullRequestEvent(w, event, azuredevopsReqID, r)
 	default:
 		e.respond(w, logging.Debug, http.StatusOK, "Ignoring unsupported event: %v %s", event.PayloadType, azuredevopsReqID)
 	}
 }
 
-// HandleGithubCommentEvent handles comment events from GitHub where Atlantis
-// commands can come from. It's exported to make testing easier.
-func (e *VCSEventsController) HandleGithubCommentEvent(event *github.IssueCommentEvent, githubReqID string, logger logging.SimpleLogging) HttpResponse {
-	if event.GetAction() != "created" {
-		return HttpResponse{
-			body: fmt.Sprintf("Ignoring comment event since action was not created %s", githubReqID),
-		}
-	}
+type EventParser interface {
+	ParsePullEvent()
+}
 
-	baseRepo, user, pullNum, err := e.Parser.ParseGithubIssueCommentEvent(event)
+type PullEvent struct {
```

Comment:
how is this used?

Reply:
Good catch, this was just scratch work when i was brainstorming. Removing.